redis/src/main/scala/com/avsystem/commons/redis/actor/RedisConnectionActor.scala

```diff
@@ -107,9 +110,13 @@ final class RedisConnectionActor(address: NodeAddress, config: ConnectionConfig)
       //TODO: use dedicated retry strategy for initialization instead of reconnection strategy
       new ConnectedTo(connection, localAddress, remoteAddress).initialize(config.reconnectionStrategy)
```

Comment:
I think we should also apply connectionId to `Connected`, we shouldn't go into initialized if `Connected` comes from different connection